content/applications/websites/website/reporting/google_analytics.rst

```diff
@@ -1,32 +1,44 @@
-=======================================================
-How to track your website's traffic in Google Analytics
-=======================================================
+=======================================
+Website analytics with Google Analytics
+=======================================
 
 To follow your website's traffic with Google Analytics:
 
-- `Create a Google Analytics account <https://www.google.com/analytics/>`__ if
-  you don't have any.
+#. `Create a Google Analytics account <https://www.google.com/analytics/>`_ if you don't have any.
+#. `Create a Google Analytics property
+   <https://support.google.com/analytics/answer/9304153?hl=en/&visit_id=638278591144564289-3612494643&rd=2>`_.
+#. Go through the property creation process and accept the Google Analytics Terms of Service
+   Agreement.
+#. When you reach the **Data collection** step, choose the platform :guilabel:`Web`.
 
-- Go through the creation form and accept the conditions to get the tracking ID.
+   .. image:: google_analytics/GA_platform.png
+      :alt: Choose a platform for your Google Analytics property.
 
-    .. image:: google_analytics/google_analytics_account.png
-        :align: center
+#. Set up your data stream: Specify your :guilabel:`Website URL` and a :guilabel:`Stream name`, then
+   click :guilabel:`Create stream`.
+#. Copy the :guilabel:`Measurement ID`.
 
-- Copy the tracking ID to insert it in Odoo.
+   .. image:: google_analytics/GA_measurement-id.png
+      :alt: Measurement ID in Google Analytics.
 
-    .. image:: google_analytics/google_analytics_tracking_id.png
-        :align: center
+#. In Odoo, go to :menuselection:`Website --> Configuration --> Settings`.
+#. In the :guilabel:`SEO` section, select the :guilabel:`Google Analytics` check box, then paste the
+   :guilabel:`Measurement ID`.
 
-- Go to the *Configuration* menu of your Odoo's Website app.
-  In the settings, turn on Google Analytics and paste the tracking ID.
-  Then save the page.
+.. note::
+   If you have :doc:`multiple websites <../configuration/multi_website>` with separate domains, it
+   is recommended to create one property per domain. To add a new property to your Google Analytics
+   account, sign in to your Google Analytics account and click the gear icon in the bottom-left
+   corner of the page to access the Admin page. Then, click :guilabel:`+ Create Property` in the
+   :guilabel:`Property` column and go through the steps described above. In Odoo, in the
+   **Website settings**, make sure to select the website in the :guilabel:`Settings of Website`
```

Comment:
The "Settings of Website" field isn't available yet in 15.0, it's only for versions 16.0 and above
![image](https://github.com/odoo/documentation/assets/95279628/71f131e6-a983-4f85-9d1f-b37d4e0bb3d7)